frontend/src/components/Comment/Comment.js

```diff
@@ -1,21 +1,42 @@
-import React from 'react'
-import "./Comment.scss";
-import {Link} from "react-router-dom";
+import React, {useState, useEffect} from 'react'
+import { Link } from "react-router-dom"
+import './SCSS/Comment.scss'
+export const Comment = (props) => {
+    const [timePosted, setTimePosted] = useState(0);
+    const [timeFormat, setTimeFormat] = useState("sec");
+    
+    useEffect(() => {
+        timeElapseSincePosted();
+    }, [])
+    const timeElapseSincePosted = () => {
```

Comment:
maybe move this function to a different file so we can re-use it (ex if we want to use the same function to display time elapsed since a picture was posted instead of a comment)

Reply:
alternatively we can maybe just find a package and import it to our project

Reply:
https://www.npmjs.com/package/javascript-time-ago